src/docs/product/error-monitoring/issue-details/index.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Issue Details
+sidebar_order: 10
+description: "Learn more about what information displays on the Issue Details page to help you efficiently triage an issue."
+---
+
+The Issue Details page helps you to gain further insight into the source of the issue and the impact it has on your application's users. The main area of the page displays information about a specific error event that's part of an issue. The right-hand sidebar of the page displays a summary of all the events grouped together in this issue.
+
+TO DO Add screenshot here of Issue Details page
+
+Click on an issue on the main Issues page to open the Issue Details page.
```

Comment:
probably unnecessary?

Reply:
I think the action of clicking is pretty intuitive, but this sentence is more about ensuring that the user understands that the page they get to when they click on an issue is called the Issue Details page, since that name isn't anywhere on the actual UI, but we use this naming convention for it.